cdn.py

```diff
@@ -8,8 +8,11 @@
 @roles('class-cache')
 def fastly_purge(*args):
     "Purge items from Fastly, eg \"/one,/two,/three\". Wildcards not supported."
-    govuk_fastly = 'http://www-gov-uk.map.fastly.net'
-    hostnames_to_purge = ['www.gov.uk', 'assets.digital.cabinet-office.gov.uk']
+    if env.environment == 'production':
+        hostnames_to_purge = ['www.gov.uk', 'assets.digital.cabinet-office.gov.uk']
+    elif env.environment == 'staging':
+        hostnames_to_purge = ['www.staging.alphagov.co.uk', 'assets.staging.alphagov.co.uk']
```

Comment:
I think we can start using the `publishing.service.gov.uk` domain suffix, even though this service hasn't been deployed yet using that hostname. In practice we use the Staging service only very infrequently for testing.